Shelfie.Identity/Shelfie.Identity.BusinessLogic/UseCases/LoginUser/LoginUserCommandHandler.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using MediatR;
+using Microsoft.AspNetCore.Identity;
+using Shelfie.Identity.BusinessLogic.Services.Interfaces;
+
+namespace Shelfie.Identity.BusinessLogic.UseCases.LoginUser;
+
+public class LoginUserCommandHandler : IRequestHandler<LoginUserCommand, string?>
+{
+    private readonly UserManager<IdentityUser> _userManager;
+    private readonly IJwtService _jwtService;
+
+    public LoginUserCommandHandler(UserManager<IdentityUser> userManager, IJwtService jwtService)
+    {
+        _userManager = userManager;
+        _jwtService = jwtService;
+    }
+
+    public async Task<string?> Handle(LoginUserCommand request, CancellationToken cancellationToken)
+    {
+        var user = await _userManager.FindByNameAsync(request.Username);
+        if (user != null && await _userManager.CheckPasswordAsync(user, request.Password))
```

Comment:
I would reverse the condition to make code more straighforward

`if (user == null || await !_userManager.CheckPasswordAsync(user, request.Password))
{ return null }`